### ISSUE TITLE
Accessibility - Add keyboard navigation

### ISSUE THREAD

**Issue by gabdube** [gabdube/native-windows-gui#81]
According to a reddit user:

> Ok, so it seems that it sort of does, but it's not quite there. Either that or the examples aren't showing it.
> The widgets are correctly seen by accessibility consumers: buttons are identified and labelled, edit boxes as well, and so on. The problem is tabbing doesn't seem to shift focus to them, so keyboard navigation is pretty much impossible.

Tab navigation can be manually implemented, but that's really annoying to do. Could probably start by added a default handler in parent widgets to move focus based on the order the control were created.

**Comment by bingen13** [comment]
Might be worth looking into how other frameworks like wxwidgets do it on win32, which works fine.

Then there are some widgets that naturally should take focus and some which shouldn't...

**Comment by gabdube** [comment]
Support added in latest PR. Tab navigation is done automatically, but the user can blacklist some control that should not receive tab focus. All tests have been updated to support keyboard navigation too.
